catroid/src/main/java/org/catrobat/catroid/content/bricks/StopScriptBrick.java

```diff
@@ -110,6 +107,7 @@ public View getPrototypeView(Context context) {
 	}
 
 	private ArrayAdapter<String> createArrayAdapter(Context context) {
+		String[] spinnerValue = new String[3];
 		spinnerValue[BrickValues.STOP_THIS_SCRIPT] = context.getString(R.string.brick_stop_this_script);
 		spinnerValue[BrickValues.STOP_ALL_SCRIPTS] = context.getString(R.string.brick_stop_all_scripts);
 		spinnerValue[BrickValues.STOP_OTHER_SCRIPTS] = context.getString(R.string.brick_stop_other_scripts);
```

Comment:
Even tough I'm not a fan of arrays and hardcoded indices, I'm leaving it as is.

Some bricks are using spinners with hard coded values like here. We should create a __single__ and future proof solution for this once and for all. What I have in mind, is, that the values may be expanded, the values are reordered or new values may be inserted in between.